Code/Framework/AzCore/AzCore/Component/EntitySerializer.cpp

```diff
@@ -81,9 +81,14 @@ namespace AZ
                     azrtti_typeid<decltype(componentMap)>(),
                     inputValue, "Components", context);
 
+            static TypeId genericComponentWrapperTypeId("{68D358CA-89B9-4730-8BA6-E181DEA28FDE}");
             for (auto& [componentKey, component] : componentMap)
```

Comment:
Have we verified that an invalid component will not be added to the container on load? Or should we still null check component?

This is not the GenericComponentWrapper case I'm thinking of. But rather regular components.

Reply:
I've tested it with EditorComponent, in this case it's not even present in componentMap as the missing element is skipped.